gobblin-core/src/main/java/org/apache/gobblin/publisher/HiveRegistrationPublisher.java

```diff
@@ -189,9 +200,27 @@ public void publishMetadata(Collection<? extends WorkUnitState> states) throws I
     // Nothing to do
   }
 
-  private static void addRuntimeHiveRegistrationProperties(State state) {
-    // Use seconds instead of milliseconds to be consistent with other times stored in hive
-    state.appendToListProp(HiveRegProps.HIVE_TABLE_PARTITION_PROPS,
-        String.format("%s:%d", DATA_PUBLISH_TIME, TimeUnit.SECONDS.convert(System.currentTimeMillis(), TimeUnit.MILLISECONDS)));
+  /**
+   * A wrapper for updating partition record count to {@link #pathToRecordCount}.
+   */
+  private void countUpForPath(String path, State state) {
```

Comment:
this function name says count, but its just getting the record count from the state, is plan to actually count the records in future? otherwise it should be renamed as getRecordCountsFromState or getRecordCounts or something.